nmdc_runtime/api/endpoints/util.py

```diff
@@ -60,6 +60,7 @@
 def list_resources(req: ListRequest, mdb: MongoDatabase, collection_name: str):
     limit = req.max_page_size
     filter_ = json_util.loads(req.filter) if req.filter else {}
+    projection = req.projection.split(",") if req.projection else None
```

Comment:
Note to self (and other reviewers):
- According to [the `find` docs](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.find), `None` is the default value of the `projection` parameter; so, when the client (i.e. request) doesn't specify a projection list, the behavior of `find` will be like it was before this change to the code.